### ISSUE TITLE
How to load different objects in different environment (used in case of parallel simulation)

### ISSUE THREAD

**Issue by v-wewei** [Genesis-Embodied-AI/Genesis#333]
I wonder how to load different objects such as different target objects for manipulation in different environment, which is common in other simulator, such as mujoco and isaacgym.  Thanks advance.

**Comment by parzival2108** [comment]
Here is how add it: Object_1 = scene.add_entity(gs.morphs.Sphere(pos =(1,1,1)))

Object_2 = scene.add_entity(gs.morphs.Box(pos =(2,2,1)))

Object_3 = scene.add_entity(gs.morphs.Cylinder(pos =(3,3,1)))

**Comment by zhouxian** [comment]
Hmm in parallel sim, every env is identical 

Indont think mujoco or Isaac support this either

**Comment by ziyanx02** [comment]
> Hmm in parallel sim, every env is identical
> 
> Indont think mujoco or Isaac support this either

I do remember isaacgym supports loading different objects in different environments. And there is also a solution in IsaacSim:
https://github.com/isaac-sim/IsaacLab/issues/186

**Comment by hgaurav2k** [comment]
This is imporant for training robust manipulation policies. I am curious if the authors plan to fix this issue. You can definitely do this in IsaacGym today. Check out [https://github.com/isaac-sim/IsaacGymEnvs/tree/main/isaacgymenvs/tasks/allegro_kuka](url)

**Comment by ArthurAllshire** [comment]
Yes, this is very important in manipulation. For an example of how it can be important check out our paper Dextrah-RGB https://dextrah-rgb.github.io/ where we train any-object grasping policy -- such efforts would be great to do in genesis but totally impossible without multi object loading

**Comment by zhouxian** [comment]
Yes this could be a useful feature. We have recently started working on adding it :) Requires a bit of work by tuning the data structure a bit

**Comment by ziqiao30** [comment]
Hey, just checking in again—any update or solution on this?

**Comment by duburcqa** [comment]
Resolved by https://github.com/Genesis-Embodied-AI/Genesis/pull/2202
